pkg/security/registry/securitycontextconstraints/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
scc files we were carrying should never have had kube copyrights... remove them when moving into origin packages